packages/platform-browser/src/browser/testability.ts

```diff
@@ -8,13 +8,17 @@
 
 import {ɵgetDOM as getDOM} from '@angular/common';
 import {GetTestability, Testability, TestabilityRegistry, ɵglobal as global} from '@angular/core';
+import {RuntimeError} from '@angular/core/src/errors';
+
+import {RuntimeErrorCode} from '../errors';
 
 export class BrowserGetTestability implements GetTestability {
   addToWindow(registry: TestabilityRegistry): void {
     global['getAngularTestability'] = (elem: any, findInAncestors: boolean = true) => {
       const testability = registry.findTestabilityInTree(elem, findInAncestors);
-      if (testability == null) {
-        throw new Error('Could not find testability for element.');
+      if (testability == null && ngDevMode) {
+        throw new RuntimeError(
+            RuntimeErrorCode.TESTABILITY_NOT_FOUND, 'Could not find testability for element.');
       }
```

Comment:
We can probably leave an error (so we don't change the logic) and just tree-shake-away the error message:
```suggestion
      if (testability == null) {
        throw new RuntimeError(
            RuntimeErrorCode.TESTABILITY_NOT_FOUND, NG_DEV_MODE && 'Could not find testability for element.');
      }
```

The `NG_DEV_MODE` can be defined in this file as:

```
const NG_DEV_MODE = typeof ngDevMode === 'undefined' || !!ngDevMode;
```